src/templates/admin/production/assign_typesetter.html

```diff
@@ -18,11 +18,11 @@
                 {% include "elements/forms/errors.html" with form=form %}
                 {% csrf_token %}
             </div>
-            {% if not article.projected_issue %}
+            {% if not article.primary_issue and not article.projected_issue %}
```

Comment:
if you want to be fully comprehensive about this, we should check for `article.issue` instead, since in some cases the article might have been added to an issue without declaring the `primary_issue` explicitly (I know, this part of the codebase has grown hairy..)